generators/openshift/files.js

```diff
@@ -10,9 +10,11 @@ function writeFiles() {
             for (let i = 0; i < this.appConfigs.length; i++) {
                 const appName = this.appConfigs[i].baseName.toLowerCase();
                 this.app = this.appConfigs[i];
-                this.template('_deployment.yml', `${this.directoryPath}/ocp/${appName}/${appName}-deployment.yml`);
+                this.template('_deployment.yml', `${this.directoryPath}/ocp/${appName}/${appName}-deployment.yml`); //microsservice
```

Comment:
There is no need to have a separate deployment file based on the nature of the app. May I know the reason behind doing this way?

Reply:
Yeah, oops. Thats a mistake. I corrected that already and put an if to check if its a monolithic application or not. If its monolithic it can be deployed via the gradle deploy task.